README.md

```diff
@@ -1 +1,163 @@
-# Metric-Wep-App
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="https://user-images.githubusercontent.com/41564330/220578014-7ed40979-15e3-491a-8c30-f764478752a5.png" width="140"  height="auto" />
+  
+  <br/>
+
+  <h3><b>Microverse</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Metricks webapp ](#-metricks-webapp-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Metricks webapp <a name="about-project"></a>
+
+> This webapp fetches the data from an api, and provided the country stats to the users. :)
+
+**[Metricks webapp]** is:
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The project is built using.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://react-bootstrap.github.io/">React Bootstrap</a></li>
+    <li><a href="https://jsx.github.io/">JSX</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node.js</a></li>
+    <li><a href="https://babeljs.io/">Babel</a></li>
+    <li><a href="https://jestjs.io/">Jest</a></li>
+  </ul>
+</details>
+
+<br>
+
+### Key Features <a name="key-features"></a>
+
+> The key features of the application are:
+
+- **[Fetches data from API]**
+- **[Displays country stats]**
+- **[Styling done using React Bootstrap]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Link to deployed project is:
+
+- [Live Demo Link] (https://luminous-dieffenbachia-b966f7.netlify.app/)
+
+> Presentation video is:
+- [Presentation Video Link] (https://www.loom.com/share/b8298de6a523498d814f0438269cecb2)
+
```

Comment:
- [ ] You did a great job recording the video and presenting the app, but the video did not meet its requirements as pointed out by the previous reviewers. Kindly follow the requirements below as you record another video.

1. The video should present the features of the project 😎
2. The video's maximum length should be 5 minutes
3. The video should contain a clear and objective description of the project in professional English 🔈
4. The video should contain a demo of the project, highlighting all the project features 💻
5. The video should contain a highlight about a piece of the code the student finds interesting or is proud of making 👍

Reply:
@elisha2kyakpo1 what was wrong with my video give more explanation, please.

Reply:
Hi @AbdifatahYasin1, the video didn't address all the requirements, for example, I didn't see the code(text editor), I didn't hear you talk about a piece of code you are proud of.